app/views/appropriate_bodies/claim_an_ect/check_ect/edit.html.erb

```diff
@@ -83,8 +83,15 @@
         )
       )
     end if @pending_induction_submission.trs_induction_status.present?
-  end
-%>
+
+  if show_extensions_row?(@teacher)
+      sl.with_row do |r| 
```

Comment:
is the tabbing slightly out here?

Reply:
yeah, it was, thanks! 

I've updated here: https://github.com/DFE-Digital/register-early-career-teachers-public/pull/508/commits/f55f4578591e0052dbb6cebf60da885b29adf3b1